packages/bsky/src/hydration/hydrator.ts

```diff
@@ -737,6 +752,33 @@ export class Hydrator {
     return { follows, followBlocks }
   }
 
+  async hydrationBidirectionalBlocks(
+    didMap: Map<string, string[]>,
+  ): Promise<BidirectionalBlocks> {
+    const pairs: RelationshipPair[] = []
+
+    for (const [uri, followers] of didMap) {
```

Comment:
How would you feel about loosening the `followers` verbiage in here?  I wonder if it may be a little confusing if you're reading this method in isolation, as it's quietly sort of alluding to the known followers feature.